OmniBLE/Bluetooth/Pair/LTKExchanger.swift

```diff
@@ -56,10 +56,10 @@ class LTKExchanger {
 
```

Comment:
Same stragety can be used to remove the force unwraps here as mentioned in PeripheralManager+Omnipod.swift.